index.html

```diff
@@ -11,188 +11,219 @@
-              width="370"
-              height="294"
-            />
-          </li>
-          <li>
-            <img
-              src="./images/picture/box-3.jpg"
-              alt="веб дізайн"
-              width="370"
-              height="294"
-            />
-          </li>
-        </ul>
+      <section class="section job">
+        <div class="container">
+          <h2 class="title-big">Чим ми займаємося</h2>
+          <ul class="list">
+            <li class="examples">
+              <img
+                src="./images/picture/box-1.jpg"
+                alt="верстка сайтів"
+                width="370"
+                height="294"
+              />
+            </li>
+            <li class="examples">
+              <img
+                src="./images/picture/box-2.jpg"
+                alt="адаптація сайтів для мобільних телефонів"
+                width="370"
+                height="294"
+              />
+            </li>
+            <li class="examples">
+              <img
+                src="./images/picture/box-3.jpg"
+                alt="веб дізайн"
+                width="370"
+                height="294"
+              />
+            </li>
+          </ul>
+        </div>
       </section>
 
       <!---Наша команда-->
-      <section class="section">
-        <h2 class="section-title title-big">Наша команда</h2>
-        <ul class="list">
-          <li>
-            <img
-              src="./images/team/ihor-demyanenko.jpg"
-              alt="фото Ігоря Дем'яненко"
-              width="270"
-              height="260"
-            />
-            <h3 class="section-title names">Ігор Дем'яненко</h3>
-            <p class="profession">Product Designer</p>
-          </li>
-          <li>
-            <img
-              src="./images/team/olha-repina.jpg"
-              alt="фото Ольги Рєпіної"
-              width="270"
-              height="260"
-            />
-            <h3 class="section-title names">Ольга Рєпіна</h3>
-            <p class="profession">Frontend Developer</p>
-          </li>
-          <li>
-            <img
-              src="./images/team/mykola-tarasov.jpg"
-              alt="фото Миколи Тарасова"
-              width="270"
-              height="260"
-            />
-            <h3 class="section-title names">Микола Тарасов</h3>
-            <p class="profession">Marketing</p>
-          </li>
-          <li>
-            <img
-              src="./images/team/mychailo-yermakov.jpg"
-              alt="фото Михайло Єрмакова"
-              width="270"
-              height="260"
-            />
-            <h3 class="section-title names">Михайло Єрмаков</h3>
-            <p class="profession">UI Designer</p>
-          </li>
-        </ul>
+      <section class="section team">
+        <div class="container">
+          <h2 class="title-big">Наша команда</h2>
+          <ul class="list">
+            <div class="person">

```

Comment:
no `div` in `ul`
And also it is unnesesary: you will need `div` fom white background on text part on the card, move bottom padding in it